src/api/urls.py

```diff
@@ -10,5 +11,7 @@
 
 app_name = "api"
 urlpatterns = [
-    path("", include(router.urls))
+    path("", include(router.urls)),
+    path("liveness/", views.liveness, name="liveness"),
+    path("readiness/", views.rediness, name="readiness"),
```

Comment:
There is a typo in the function name `rediness`. It should be `readiness` to match the function defined in `views.py`.